src/storage/store/rel_table_data.cpp

```diff
@@ -88,7 +78,6 @@ void RelDataReadState::resetState() {
     numNodes = 0;
     currentNodeOffset = 0;
     posInCurrentCSR = 0;
-    csrListEntries.resize(StorageConstants::NODE_GROUP_SIZE, {0, 0});
     csrHeaderChunks = ChunkedCSRHeader(false);
```

Comment:
Now revisit this, I think we can just reset offset and length chunks by `resetEmpty()`, instead of re-creating new chunks?

Reply:
Done. I also removed the resetState call in the constructor. The only thing that the constructor wasn't doing was resizing the csrListEntries, so now it's entirely redundant.